pkg/doc/jwt/claims.go

```diff
@@ -0,0 +1,123 @@
+/*
```

Comment:
These are specifically JWT registered claims (aka JWT's data model). This file doesn't actually handle all claims in the JWT.

Reply:
These are the base claims. A user can extend `Claims` (e.g. via struct embedding). Or you mean we need to extend the `Claims` itself with some extra fields? I added the fields based on the https://tools.ietf.org/html/rfc7519#section-4.1.